docs/bastion.md

```diff
@@ -75,11 +75,30 @@ The bastion is accessed via an AWS ELB. The ELB is required to gain secure acces
 
 You can increase the ELB idle timeout by editing the main cluster config `kops edit cluster $NAME` and modifyng the following block
 
-```
+```yaml
 spec:
   topology:
     bastion:
       idleTimeoutSeconds: 1200
 ```
 
 Where the maximum value is 1200 seconds (20 minutes) allowed by AWS. [More information](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/config-idle-timeout.html)
+
+### Using the bastion
+
+Once your cluster is setup and you need to SSH into the bastion you can access a cluster resource using the following steps
+
+```bash
+# Verify you have an SSH agent running. This should match whatever you built your cluster with.
+ssh-add -l 
+# If you need to add an agent
+ssh-add path/to/public/key
+
+# Now you can SSH into the bastion
+ssh -A admin@<bastion-ELB-address>
+
+# Where <bastion-ELB-address> is usually bastion.$clustername (bastion.example.kubernetes.cluster) unless otherwise specified
+
+```
+
+Now that you can successfully SSH into the bastion with a forwarded SSH agent. You can SSH into any of your cluster resources using their local IP address. You can get their local IP address from the cloud console.
```

Comment:
Or using `kubectl get nodes` or (soon) `kops toolbox dump`.  Sadly both are still awkward.  But this LGTM!